netflix/ribbon-secured/src/test/java/org/wildfly/swarm/ts/netflix/ribbon/secured/RibbonSecuredTopologyIT.java

```diff
@@ -173,8 +176,7 @@ public void accessThroughRibbon_withToken_serviceDisabled() throws Exception {
 
         String token = authzClient.obtainAccessToken("test-user", "test-password").getToken();
 
-        String response = Request.Get("http://localhost:8080/test")
-                .addHeader("Authorization", "Bearer " + token)
+        String response = Request.Get("http://localhost:8080/test").addHeader("Authorization", "Bearer " + token)
```

Comment:
I don't understand the reformatting in this file.

Reply:
I did with Eclipse, will fix